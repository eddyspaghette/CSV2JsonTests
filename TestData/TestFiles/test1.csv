1,100,200,300,400,500
2,101,201,301,401,501
3,102,202,302,,502
4,103,203,303,403,503
5,104,204,304,404,504
6,105,,305,405,505
7,106,206,306,406,506
8,,207,307,407,507
9,108,208,308,408,508
10,109,209,309,409,509
11,110,210,310,410,510
12,111,211,311,411,511
13,,212,312,412,512
14,113,213,313,413,513
15,114,214,314,414,514
16,115,215,315,415,
17,116,216,316,,516
18,117,217,317,417,517
19,118,218,318,418,518
20,119,219,319,419,519
