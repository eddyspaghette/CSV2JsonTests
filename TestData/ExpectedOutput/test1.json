[
 {
   "FIELD1": 1,
   "FIELD2": 100,
   "FIELD3": 200,
   "FIELD4": 300,
   "FIELD5": 400,
   "FIELD6": 500
 },
 {
   "FIELD1": 2,
   "FIELD2": 101,
   "FIELD3": 201,
   "FIELD4": 301,
   "FIELD5": 401,
   "FIELD6": 501
 },
 {
   "FIELD1": 3,
   "FIELD2": 102,
   "FIELD3": 202,
   "FIELD4": 302,
   "FIELD5": null,
   "FIELD6": 502
 },
 {
   "FIELD1": 4,
   "FIELD2": 103,
   "FIELD3": 203,
   "FIELD4": 303,
   "FIELD5": 403,
   "FIELD6": 503
 },
 {
   "FIELD1": 5,
   "FIELD2": 104,
   "FIELD3": 204,
   "FIELD4": 304,
   "FIELD5": 404,
   "FIELD6": 504
 },
 {
   "FIELD1": 6,
   "FIELD2": 105,
   "FIELD3": null,
   "FIELD4": 305,
   "FIELD5": 405,
   "FIELD6": 505
 },
 {
   "FIELD1": 7,
   "FIELD2": 106,
   "FIELD3": 206,
   "FIELD4": 306,
   "FIELD5": 406,
   "FIELD6": 506
 },
 {
   "FIELD1": 8,
   "FIELD2": null,
   "FIELD3": 207,
   "FIELD4": 307,
   "FIELD5": 407,
   "FIELD6": 507
 },
 {
   "FIELD1": 9,
   "FIELD2": 108,
   "FIELD3": 208,
   "FIELD4": 308,
   "FIELD5": 408,
   "FIELD6": 508
 },
 {
   "FIELD1": 10,
   "FIELD2": 109,
   "FIELD3": 209,
   "FIELD4": 309,
   "FIELD5": 409,
   "FIELD6": 509
 },
 {
   "FIELD1": 11,
   "FIELD2": 110,
   "FIELD3": 210,
   "FIELD4": 310,
   "FIELD5": 410,
   "FIELD6": 510
 },
 {
   "FIELD1": 12,
   "FIELD2": 111,
   "FIELD3": 211,
   "FIELD4": 311,
   "FIELD5": 411,
   "FIELD6": 511
 },
 {
   "FIELD1": 13,
   "FIELD2": null,
   "FIELD3": 212,
   "FIELD4": 312,
   "FIELD5": 412,
   "FIELD6": 512
 },
 {
   "FIELD1": 14,
   "FIELD2": 113,
   "FIELD3": 213,
   "FIELD4": 313,
   "FIELD5": 413,
   "FIELD6": 513
 },
 {
   "FIELD1": 15,
   "FIELD2": 114,
   "FIELD3": 214,
   "FIELD4": 314,
   "FIELD5": 414,
   "FIELD6": 514
 },
 {
   "FIELD1": 16,
   "FIELD2": 115,
   "FIELD3": 215,
   "FIELD4": 315,
   "FIELD5": 415,
   "FIELD6": null
 },
 {
   "FIELD1": 17,
   "FIELD2": 116,
   "FIELD3": 216,
   "FIELD4": 316,
   "FIELD5": null,
   "FIELD6": 516
 },
 {
   "FIELD1": 18,
   "FIELD2": 117,
   "FIELD3": 217,
   "FIELD4": 317,
   "FIELD5": 417,
   "FIELD6": 517
 },
 {
   "FIELD1": 19,
   "FIELD2": 118,
   "FIELD3": 218,
   "FIELD4": 318,
   "FIELD5": 418,
   "FIELD6": 518
 },
 {
   "FIELD1": 20,
   "FIELD2": 119,
   "FIELD3": 219,
   "FIELD4": 319,
   "FIELD5": 419,
   "FIELD6": 519
 }
]